### PR TITLE
Draft of Certbot snap announcement

### PR DESCRIPTION
**Pull request by bmw**
This PR should never be merged, however, the file here is my proposed text for announcing the Certbot snap beta on https://community.letsencrypt.org/, the Certbot mailing list, etc. Actually posting this text is blocked on https://github.com/certbot/website/pull/558.

If people have any feedback, I'd love to hear it!

### CODE REVIEW

File: snap-announcement.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# A New Way to Install Certbot on Linux
+
+Today we released Certbot packaged as a snap offering a new way to install the client. Most modern Linux distributions (basically any that use systemd) can use the Certbot snap. Some of the benefits of installing Certbot this way are:
+
+* Certbot automatically stays up-to-date giving you access to the latest features including updates to the TLS configuration Certbot uses when installing certificates with Apache and Nginx.
```
Comment (schoen) [inline snap-announcement.md:5]: I suggest a comma: "stays up-to-date, giving you access to"

File: snap-announcement.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+* Automatic renewal comes preconfigured so there is no need to manually set up a cron job or systemd timer.
```
Comment (schoen) [inline snap-announcement.md:6]: I'd also suggest "comes preconfigured, so there is no need"

File: snap-announcement.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# A New Way to Install Certbot on Linux
+
+Today we released Certbot packaged as a snap offering a new way to install the client. Most modern Linux distributions (basically any that use systemd) can use the Certbot snap. Some of the benefits of installing Certbot this way are:
```
Comment (schoen) [inline snap-announcement.md:3]: "as a snap, offering a new way to install in the client"

File: snap-announcement.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# A New Way to Install Certbot on Linux
+
+Today we released Certbot packaged as a snap, offering a new way to install the client. Most modern Linux distributions (basically any that use systemd) can use the Certbot snap. Some of the benefits of installing Certbot this way are:
+
+* Certbot automatically stays up-to-date, giving you access to the latest features including updates to the TLS configuration Certbot uses when installing certificates with Apache and Nginx.
+* Automatic renewal comes preconfigured, so there is no need to manually set up a cron job or systemd timer.
+* While not initially available, we're planning to release snaps for all of our DNS plugins as well as instructions for people to create their own 3rd party plugin snaps in the coming months.
+
+Support for Certbot packaged as a snap is currently in its beta phase and only supports the x86_64 architecture, but if you'd like to help us test it, you can find instructions for installing the Certbot snap at
+https://certbot.eff.org/instructions by selecting your server software and then choosing "snapd" in the "System" dropdown menu.
```
Comment (ohemorange) [inline snap-announcement.md:10]: "(select "snapd" regardless of your operating system, as instructions are the same across operating systems)"

Comment (bmw) [reply snap-announcement.md]: I tweaked the wording slightly, but added something close to that.

Comment (ohemorange) [reply snap-announcement.md]: love it!

**Comment by bmw** [comment]
I posted this at https://community.letsencrypt.org/t/a-new-way-to-install-certbot-on-linux/120408.

Thanks a lot for the reviews!

**Pull request closed without merging**